src/VCR.ts

```diff
@@ -62,10 +62,13 @@ async function writeCassetteFile(cassettes: CassetteFile): Promise<void> {
 async function saveCassettes(defns: nock.Definition[]): Promise<void> {
   const cassettes: CassetteFile = await readCassetteFile()
   const testName = expect.getState().currentTestName
-  if (testName) {
-    cassettes[testName] = defns
-    await writeCassetteFile(cassettes)
-  }
+  if (!testName) return
```

Comment:
I don't know. That condition was there, so I didn't change it.
I'm guessing it's to prevent someone from accidentally putting this into the normal code

Reply:
ok, wil resolve.